### ISSUE TITLE
Support stock for listing

### ISSUE THREAD

**Issue by morrissimo** [Xwaffle1/UniversalMarket#2]
Current version (at least for 1.10.2) only supports the items in the actual listing itself. Would like to see support for a stock or supply of items beyond the listing itself, so that multiple purchases could be made off of a single listing. When the stock is depleted, the listing would disappear (per the current functionality)

**Comment by Xwaffle1** [comment]
Makes sense to allow people to partially purchase an item. I'll look into the concept, should be able to incorporate this :)
